lib/src/meteor_client.dart

```diff
@@ -326,29 +330,36 @@ class MeteorClient {
   /// A Map containing user documents.
   Stream<Map<String, dynamic>> get users => collections['users'];
 
-  /// True if a login method (such as Meteor.loginWithPassword, Meteor.loginWithFacebook, or Accounts.createUser) is currently in progress.
+  /// Current log-in status of login methods (such as Meteor.loginWithPassword, Meteor.loginWithFacebook, or Accounts.createUser).
   /// A reactive data source.
-  Stream<bool> loggingIn() {
-    return _loggingInStream;
+  Stream<UserLogInStatus> logInStatus() {
+    return _logInStatusStream;
   }
 
+  /// Current log-in status if user is logging in.
+  bool loggingIn() => _logInStatus == UserLogInStatus.loggingIn;
+  
+  /// Current log-out status if user is logging out.
+  bool loggingOut() => _logInStatus == UserLogInStatus.loggingOut;
```

Comment:
Hi @tanutapi , 
Added the two methods as following the original Meteor APIs, 
- loggingIn - https://github.com/meteor/meteor/blob/master/packages/accounts-base/accounts_client.js#L740
- loggingOut - https://github.com/meteor/meteor/blob/master/packages/accounts-base/accounts_client.js#L748

And as you can see it is determined by the `_logInStatus` which now we can through `logInStatus()` to get the Stream and listening the loggingIn and loggingOut status as well.